gui/src/test/resources/org/verapdf/processor/bad-profile.xml

```diff
@@ -8,7 +8,7 @@
     <rules>
```

Comment:
Everything from here onwards except the POM are changes to whitespace brought about by your IDE and they shouldn't be in the PR, neither should the POM for that matter.